src/ape/api/accounts.py

```diff
@@ -209,7 +209,9 @@ def check_signature(
                     "Parameter 'signature' required when verifying a 'SignableMessage'."
                 )
         elif isinstance(data, TransactionAPI):
-            return self.address == Account.recover_transaction(data.serialize_transaction())
+            return self.address == Account.recover_transaction(
+                data.serialize_transaction()  # type: ignore
```

Comment:
Is this type ignore needed? I think `recover_transaction()` should be able to accept bytes